### PR TITLE
Outagepredict migration

### PR DESCRIPTION
**Pull request by rvelaVenafi**
Migrated from devOps Accelerate to OutagePredict API.
Updated test suite to match API requirements.
Disabled Cucumber test related to CSR reuse, as this scenario is not supported yet.

### CODE REVIEW

**Review by marcos-albornoz — APPROVED** [review]

File: pkg/venafi/cloud/cloud.go [hunk 1]
```diff
@@ -79,18 +75,23 @@ type certificateRequestClientInfo struct {
...
+	ReuseCSR                 bool                         `json:"reuseCSR,omitempty"`
+	ValidityPeriod           string                       `json:"validityPeriod,omitempty"`
 }
 
 type certificateStatus struct {
-	Id                        string                            `json:"Id,omitempty"`
-	ManagedCertificateId      string                            `json:"managedCertificateId,omitempty"`
-	ZoneId                    string                            `json:"zoneId,omitempty"`
+	Id string `json:"id,omitempty"`
+	//ManagedCertificateId      string                            `json:"managedCertificateId,omitempty"`
```
Comment (angelmoo) [inline pkg/venafi/cloud/cloud.go:91]: I think we can remove this, since it will not be used.

File: pkg/venafi/cloud/connector.go [hunk 2]
```diff
@@ -537,16 +544,16 @@ func (c *Connector) searchCertificatesByFingerprint(fp string) (*CertificateSear
...
 */
 type managedCertificate struct {
-	Id                         string `json:"id"`
-	CompanyId                  string `json:"companyId"`
-	LatestCertificateRequestId string `json:"latestCertificateRequestId"`
-	CertificateName            string `json:"certificateName"`
+	Id                   string `json:"id"`
+	CompanyId            string `json:"companyId"`
+	CertificateRequestId string `json:"certificateRequestId"`
+	//CertificateName            string `json:"certificateName"`
```
Comment (angelmoo) [inline pkg/venafi/cloud/connector.go:550]: can we safely remove this commented code?

File: pkg/venafi/cloud/cloud.go (hunk 1 above)
Comment (rvelaVenafi) [reply pkg/venafi/cloud/cloud.go:91]: done

**Review by angelmoo — APPROVED** [review]
changes looks good

File: pkg/venafi/cloud/connector.go (hunk 2 above)
Comment (rvelaVenafi) [reply pkg/venafi/cloud/connector.go]: done

**Pull request merged**